ui/scss/component/_claim-list.scss

```diff
@@ -970,24 +970,26 @@
     }
   }
 
-  &:hover {
-    cursor: pointer;
+  @media (hover) {
```

Comment:
I think the issue also happens in list layout. 
 When we have a solution that confines to Safari, can you verify that it also covers list layout?